core-web/libs/template-builder/src/lib/components/template-builder/components/dot-layout-properties/dot-layout-properties-item/dot-layout-properties-item.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, EventEmitter, forwardRef, HostListener, Input, Output } from
 import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 
 @Component({
+    // eslint-disable-next-line @angular-eslint/component-selector
```

Comment:
this is because in the lib is a lint rule that the selectors should start with `dotcms` I didn't change it so it doesn't break in the other sections of the project that this component it being used. So, I just disable the component selector rule in that component